src/PrestaShopBundle/Resources/config/services/core/command_bus.yml

```diff
@@ -0,0 +1,13 @@
+services:
+    _defaults:
+        public: true
+
+    prestashop.core.command_bus.tactician_command_bus_adapter:
```

Comment:
I know the rule is generally to follow the namespace, but since this will potentially be used in many many places, what do you think about `prestashop.core.command_bus` and `prestashop.core.query_bus`?

Reply:
makes sense, done. :+1: